.circleci/config.yml

```diff
@@ -156,3 +156,40 @@ workflows:
     jobs:
       - test-android
       - deploy-android: *release-tags
+  
+  android-deploy-on-tag:
+    jobs:
+      - test-android:
+          filters:
+            tags:
+              only: /^(\d+\.)(\d+\.)(\d+)$/
+            branches:
+              ignore: /.*/
+      - deploy-android:
+          requires:
+            - test-android
+          filters:
+            tags:
+              only: /^(\d+\.)(\d+\.)(\d+)$/
+            branches:
+              ignore: /.*/
```

Comment:
how does this differ from the `deploy-android` job?

Reply:
we don't need these, removing. I think they are leftovers from manually triggered releases we've been doing in the past year